lib/person.rb

```diff
@@ -1,4 +1,6 @@
+require_relative 'nameable'
 class Person
+  include Nameable
```

Comment:
- [ ] It would be great if after making the Nameable a class you make this Person Class a child class of the Nameable class to follow the project requirements and decorator pattern 👍